scripts/create-package.sh

```diff
@@ -0,0 +1,243 @@
+        -c          : Git commit
+        -b          : Git branch
+        -r          : Git repository
+        -v          : dosbox-staging version
+        BUILD_DIR   : Meson build directory
+        PACKAGE_DIR : Package directory
+    
+    Note: On macos, '-v' must be set. On msvc, the environment variable VC_REDIST_DIR must be set."
+}
+
+create_parent_dir()
+{
+    path=$1
+    dir=$(dirname "$path")
+    if [ "$dir" != "." ]; then
+        case $platform in
+             msvc) mkdir -p "$dir" ;;
+             macos) install -d "${dir}/"
+        esac
+    fi
+}
+
+install_file()
+{
+    src=$1
+    dest=$2
+    case $platform in
+        linux|msys2) install -DT -m 644 "$src" "$dest" ;;
+        msvc) create_parent_dir "$dest" && cp "$src" "$dest" ;;
+        macos) create_parent_dir "$dest" && install -m 644 "$src" "$dest" ;;
+    esac
+}
+
+install_doc()
+{
+    # Install common documentation files
+    case $platform in
+        linux)
+            install_file docs/README.template "${pkg_dir}/README"
+            install_file COPYING              "${pkg_dir}/COPYING"
+            install_file README               "${pkg_dir}/doc/manual.txt"
+            install_file docs/dosbox.1        "${pkg_dir}/man/dosbox.1"
+            readme_tmpl="${pkg_dir}/README"
+            ;;
+        macos)
+            install_file docs/README.template "${macos_dst_dir}/SharedSupport/README"
+            install_file COPYING              "${macos_dst_dir}/SharedSupport/COPYING"
+            install_file README               "${macos_dst_dir}/SharedSupport/manual.txt"
+            install_file docs/README.video    "${macos_dst_dir}/SharedSupport/video.txt"
+            readme_tmpl="${macos_dst_dir}/SharedSupport/README"
+            ;;
+        msys2|msvc)
+            install_file COPYING              "${pkg_dir}/COPYING.txt"
+            install_file docs/README.template "${pkg_dir}/README.txt"
+            install_file docs/README.video    "${pkg_dir}/doc/video.txt"
+            install_file README               "${pkg_dir}/doc/manual.txt"
+            readme_tmpl="${pkg_dir}/README.txt"
+            ;;
+    esac
+    # Fill template variables in README.template
+    [ -n "$git_commit" ] && sed -i -e "s|%GIT_COMMIT%|$git_commit|" "$readme_tmpl"
+    [ -n "$git_branch" ] && sed -i -e "s|%GIT_BRANCH%|$git_branch|" "$readme_tmpl"
+    [ -n "$git_repo" ]   && sed -i -e "s|%GITHUB_REPO%|$git_repo|"  "$readme_tmpl"
+}
+
+install_translation()
+{
+    lng_dir=${pkg_dir}/translations
+    if [ "$platform" = "macos" ]; then
+        lng_dir=${macos_dst_dir}/Resources/translations
+    fi
+    # Prepare translation files
+    #
+    # Note:
+    #   We conciously drop the dialect postfix because no dialects are available.
+    #   (US was the default DOS dialect and therefore is the default for 'en').
+    #   There users get the generic translation and benefit from simpler filenames.
```

Comment:
` #   There users get the generic translation and benefit from simpler filenames.`
(suggest removing this line..  I had meant to say "therefore users benefit from simpler filenames"; but it's kind of extraneous.  Doesn't add much value.)